cmd/packet_capture.go

```diff
@@ -146,8 +140,8 @@ func runPacketCaptureOnAddr(port int, filename string) {
 			// write enriched data as interface
 			writeEnrichedData(pw, &genericMap)
 
-			// then append packet to file
-			err = pw.WriteEnhancedPacketBlock(0, ts, b, types.EnhancedPacketOptions{})
+			// then append packet to file using totalPackets as unique id
+			err = pw.WriteEnhancedPacketBlock(totalPackets, ts, b, types.EnhancedPacketOptions{})
```

Comment:
@jpinsonneau how the total packet can be unique id ?

Reply:
I increase the `totalPackets` value each time we add a packet just below this call:
```
		totalPackets = totalPackets + 1
```

Reply:
yeah I saw that isn't that will be the case with other pkts u capture so if I received 2 udp pkts then 2 tcp pkts in two different runs won't that collide ?

Reply:
do you open the 2 pcapng files in one ? :thinking:

Reply:
I tried the following code to see what happens:
```
err = pw.WriteEnhancedPacketBlock(uint32(startupTime.UnixMilli())+totalPackets, ts, b, types.EnhancedPacketOptions{})
```

And it seems wireshark is not expecting that. Here the id is an index related to the previously created interface description using `WriteInterfaceDescription` call:
![image](https://github.com/user-attachments/assets/8ac943ab-7766-4c03-889e-858f8aba93d9)

Reply:
ok I guess lets proceed as is thanks for confirming